os/net/blemgr/bledev.c

```diff
@@ -84,32 +84,40 @@ int bledev_handle(struct bledev *dev, lwnl_req cmd, void *data, uint32_t data_le
 		TRBLE_DRV_CALL(ret, dev, disconnect, (dev, con_handle, mode));
 	}
 	break;
-	case LWNL_REQ_BLE_DEL_BOND:
+	case LWNL_REQ_BLE_GET_BONDED_DEV:
 	{
-		trble_bd_addr *addr = NULL;
-		trble_mode_e mode;
+		trble_bonded_device_list_s *device_list = NULL;
+		uint16_t *device_count = NULL;
 
 		lwnl_msg_params param = { 0, };
 		if (data != NULL) {
 			memcpy(&param, data, data_len);
 		} else {
 			return TRBLE_INVALID_ARGS;
 		}
-		addr = (trble_bd_addr *)param.param[0];
-		mode = *(trble_mode_e *)param.param[1];
+		device_list = (trble_bonded_device_list_s *)param.param[0];
+		device_count = (uint16_t *)param.param[1];
 
-		TRBLE_DRV_CALL(ret, dev, del_bond, (dev, addr, mode));
+		TRBLE_DRV_CALL(ret, dev, get_bonded_dev, (dev, device_list, device_count));
```

Comment:
`device_count` is pointer type. Most of other `TRBLE_DRV_CALL`, it passes the int variable.
Is it okay to pass the pointer type?

Reply:
'device_count' is for getting the number of bonded devices.  To do that, this should be pointer